service/history/workflowResetor.go

```diff
@@ -776,11 +778,19 @@ func (w *workflowResetorImpl) ApplyResetEvent(
 		return retError
 	}
 	defer func() {
-		w.eng.historyV2Mgr.CompleteForkBranch(&persistence.CompleteForkBranchRequest{
+		err := w.eng.historyV2Mgr.CompleteForkBranch(&persistence.CompleteForkBranchRequest{
```

Comment:
we actually don't want to return error even completeFork failed. There will be a background jog takes care of it. For this error let's simply log it.